adapters/appnexus.go

```diff
@@ -235,3 +237,42 @@ func NewAppNexusAdapter(config *HTTPAdapterConfig, externalURL string) *AppNexus
 		usersyncInfo: info,
 	}
 }
+
+func CollectImpData(oreq openrtb.BidRequest, req *pbs.PBSRequest, unit pbs.PBSAdUnit) string {
+	data := ""
+
+	//set 0 for web impression, and 1 for mobile impression
+	if oreq.Site != nil {
+		data += "plat:0"
```

Comment:
I think generally string formatting functions are prefered vs raw concatenation. For example `fmt.Sprintf` I think would be appropriate here.